.github/workflows/generate_exe.yml

```diff
@@ -0,0 +1,24 @@
+name: Package exe with PyInstaller - Windows
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: PyInstaller Windows
+      uses: JackMcKew/pyinstaller-action-windows@main
+      with:
+        path: ./
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: AutoStructureFilesForDFplayer
+        path: ./dist/windows # or path/to/artifact
```

Comment:
As I'd like to have the executable in one file only, please add `--onefile` to the pyinstaller command

Reply:
this spec file is generated with the `--one-file` option.  

https://pyinstaller.readthedocs.io/en/stable/spec-files.html for reference.